src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts

```diff
@@ -83,8 +83,8 @@ export class UserListComponent implements OnInit {
         this.modalRef.hide();
         this.notificationService.show(
           NotificationType.success,
-          `User "${username}" has been deleted.`,
-          'Delete User'
+          'Delete user "${username}"',
+          `User "${username}" has been deleted.`
```

Comment:
This argument is now only used for error message descriptions. Please remove this line.